Q/Qhull/build_tarballs.jl

```diff
@@ -55,6 +55,20 @@ products = [
     # Libraries
     # reentrant Qhull
     LibraryProduct(["libqhull_r", "qhull_r"], :libqhull_r),
+    
+    # Files
+    # Reentrant Qhull header files
+    FileProduct("include/libqhull_r/geom_r.h", :geom_r),
+    FileProduct("include/libqhull_r/io_r.h", :io_r),
+    FileProduct("include/libqhull_r/libqhull_r.h", :libqhull_r),
```

Comment:
Doesn't this have the same name as the library product above?

Reply:
whoops, it does.  I'll change all of them to `*_r_h`...